test_tor_mirror_repo_declared_only_once failing on release/0.9

## Description

CI is failing on the latest release branch

## Steps to Reproduce

Look at all CI builds on `release/0.9`: https://circleci.com/gh/freedomofpress/securedrop/tree/release%2F0%2E9

## Expected Behavior

Builds are green

## Actual Behavior

The staging CI [job](https://circleci.com/gh/freedomofpress/securedrop/17370) [is](https://circleci.com/gh/freedomofpress/securedrop/17349) [consistently](https://circleci.com/gh/freedomofpress/securedrop/17266) [failing](https://circleci.com/gh/freedomofpress/securedrop/17249):

```
    ________ test_tor_mirror_repo_declared_only_once[ansible://app-staging] ________
    [gw19] linux2 -- Python 2.7.13 /usr/bin/python
    
    host = <testinfra.host.Host object at 0x7fa730986650>
    
        def test_tor_mirror_repo_declared_only_once(host):
            """
            The apt repo config is written both via the `securedrop-config`
            package and via Ansible at install time. The filename logic
            was slightly different for both between 0.5 and 0.6, so let's
            ensure that we've cleaned up adequately, and the tor-apt mirror
            is declared on the system only once.
            """
            c = host.command("grep -rl tor-apt /etc/apt/")
            assert c.rc == 0
            files_found = c.stdout.rstrip("\n").split("\n")
            # Two files are expected, cron-apt and one apt repo
    >       assert len(files_found) == 2
    E       AssertionError: assert 1 == 2
    E        +  where 1 = len(['/etc/apt/security.list'])
    
    ../testinfra/staging/common/test_tor_mirror.py:95: AssertionError
    ________ test_tor_mirror_repo_declared_only_once[ansible://mon-staging] ________
    [gw30] linux2 -- Python 2.7.13 /usr/bin/python
    
    host = <testinfra.host.Host object at 0x7f3d39d09410>
    
        def test_tor_mirror_repo_declared_only_once(host):
            """
            The apt repo config is written both via the `securedrop-config`
            package and via Ansible at install time. The filename logic
            was slightly different for both between 0.5 and 0.6, so let's
            ensure that we've cleaned up adequately, and the tor-apt mirror
            is declared on the system only once.
            """
            c = host.command("grep -rl tor-apt /etc/apt/")
            assert c.rc == 0
            files_found = c.stdout.rstrip("\n").split("\n")
            # Two files are expected, cron-apt and one apt repo
    >       assert len(files_found) == 2
    E       AssertionError: assert 1 == 2
    E        +  where 1 = len(['/etc/apt/security.list'])
    
    ../testinfra/staging/common/test_tor_mirror.py:95: AssertionError
```

--- a/molecule/aws/tor_apt_test.yml
+++ b/molecule/aws/tor_apt_test.yml
@@ -4,19 +4,6 @@
     data: "{{ lookup('file','securedrop_test.pub') }}"
     state: present
 
-- name: Temporary fix for GH issue 2938
-  file:
-    state: absent
-    path: "/etc/apt/sources.list.d/tor_apt_freedom_press.list"
-
-- name: Switch apt repo URLs to staging.
-  replace:
-    dest: "/etc/apt/sources.list.d/tor.apt.freedom.press.list"
-    replace: "tor-apt-test.freedom.press"
-    regexp: '//tor-apt\.freedom\.press'
-  ignore_errors: "yes"
-  notify: update tor
-
 - name: Force possible tor update
   meta: flush_handlers
 

